nilearn/plotting/displays.py

```diff
@@ -1089,7 +1093,8 @@ def add_graph(self, adjacency_matrix, node_coords,
                   edge_cmap=cm.bwr,
                   edge_vmin=None, edge_vmax=None,
                   edge_threshold=None,
-                  edge_kwargs=None, node_kwargs=None):
+                  edge_kwargs=None, node_kwargs=None, colorbar=False,
+                  ):
```

Comment:
@lesteve : I disagree: such a layout makes it easy to add new arguments. It's actually good practice to do this in Python.